### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Surfsup1408**
Please fix my mess

### CODE REVIEW

**Comment by Gibtsawit** [comment]
lets hope we can see this comment

**Review by AshleeValle — APPROVED** [review]
after all the trouble.... thumbs up.

**Comment by ehteshama12** [comment]
> lets hope we can see this comment

oh I definitely see it

**Review by ehteshama12 — APPROVED** [review]
Quite aggressive, don't ya think?

**Review by Gibtsawit — APPROVED** [review]
got it!

File: src/team/g6/DemoG6.java [hunk 1]
```diff
@@ -65,6 +65,8 @@ public static void goodBye() { //LOCAL
 			// just another test
 	
 		System.out.println("New Edits for Review");
+		
+		System.out.println("It's a mad world");
```
Comment (AshleeValle) [inline src/team/g6/DemoG6.java:69]: yes it is

Comment (Eddyabdy) [reply src/team/g6/DemoG6.java:69]: which one?

**Comment by Eddyabdy** [comment]
> Please fix my mess
I fixed it

**Pull request merged**